index.js

```diff
@@ -38,8 +38,8 @@ module.exports = function(content) {
 	var outputPath = "";
 
 	var filePath = this.resourcePath;
+	var issuerContext = this._module && this._module.issuer && this._module.issuer.context || context;
```

Comment:
@michael-ciniawsky Well... Actually this line of code is from the `master` branch. I just moved it outside the block. Do you mean I should choose `issuer` or `context` to pass to the `publicPath` function?